src/promptflow-azure/promptflow/azure/_storage/cosmosdb/summary.py

```diff
@@ -74,14 +75,15 @@ def __init__(self, span: Span, collection_id: str, created_by: typing.Dict, logg
         self.span = span
         self.created_by = created_by
         self.logger = logger
+        self.session_id = self.span.resource.get(SpanResourceAttributesFieldName.COLLECTION, TRACE_DEFAULT_COLLECTION)
```

Comment:
Is this the new session_id concept in chat window scenario?  why this get from collection?

Reply:
no, this is cloud scenario, old session id, corresponding to our collection concept for now; i do not change this var name to avoid more change in this kind of huge pr.

Reply:
cc. @huaiyan and @riddlexu